### ISSUE TITLE
The value of a Less variable declared with no whitespace in front of its colon is read by nobody, its first word being welded into the at-rule name

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#649]
### What minimal example or steps are needed to reproduce the bug?

A Less variable declared with no whitespace in front of its colon:

```less
@v:10PX 1px;
b { c: @v }
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "overrides": [
    {
      "files": ["**/*.less"],
      "customSyntax": "postcss-less",
      "rules": {
        "@stylistic/less/unit-case": "lower"
      }
    }
  ]
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.less" --config myconfig.json
```

### What did you expect to happen?

One warning under `lower`, about the `PX` and about nothing else, and `--fix` writing `@v:10px 1px;`. Less declares `@v` here as it does on every other spelling of the whitespace around the colon: `less@4.9.1` compiles that file to `b { c: 10PX 1px; }`, so `10PX 1px` is the value and `PX` a unit of it.

### What actually happened?

Nothing under either option: no warning, and `--fix` writes nothing. `@v:10PX;`, whose whole value is that first word, draws nothing either.

### What is the cause?

Where no whitespace and no comment stands in front of the colon, `postcss-less@6.0.0` reads `@v:10PX` as one at-word and puts it in the node's **name**: the node comes back with `name` `v:10PX` and `params` `1px`, where `@v : 10PX 1px` comes back with `name` `v` and `params` `": 10PX 1px"`. Every rule of this plugin that reads an at-rule's text reads `params`, so the first word of such a value is read by nobody — and what is left there is no value of its own, `@v:url(10PX)` coming back named `v:url` with the nameless group `(10PX)` for params, so the whole declaration is passed over rather than read in halves. It is the other half of [#577](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/577), which put the reading of that node on its shape rather than on the parser's `variable` mark and so reached the params wherever they open on the colon; this half is not about which nodes the rule walks but about where the parser put the text.

### Do you have a proposal to fix the bug?

Not one that is obviously right, and that is why this is filed apart. The value is contiguous in the printed node — `name.slice(colonIndex + 1)` runs through `raws.afterName` into `params` — so a reading could hand a rule that whole text and split a fix back at the same boundary. But `read` and `write` of the `Syntax` contract are shared by every rule that touches an at-rule's params, and widening them there would change what all of those read; a reading for this one rule would be a contract member of its own. Measure first what the fix would have to write into `name`, and whether `at-rule-name-case`, the only rule that writes a name, can be made to leave that tail alone under every option.
